src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.form__big-search-bar {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 21.86px;
+  color: rgba(61, 78, 97, 1);
+  padding: 25px 0 26px 62px;
+  margin: 12px 0 20px;
+  outline: none;
+  border: 1px solid rgba(225, 231, 237, 1);
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)